examples/patterns/grid/nested.html

```diff
@@ -3,29 +3,265 @@
       <div class="row">
-        <div class="col-3">
-          <span>.col-3</span>
+        <div class="col-small-3 col-medium-3 col-9">
+          col-small-3 .col-medium-3 .col-9
           <div class="row">
-            <div class="col-1">
-              <span>.col-1</span>
+            <div class="col-small-1 col-medium-1 col-2">
+              col-small-1 col-medium-1 col-2
             </div>
-            <div class="col-1">
-              <span>.col-1</span>
+            <div class="col-small-1 col-medium-1 col-3">
+              col-small-1 col-medium-1 col-2
             </div>
-            <div class="col-1">
-              <span>.col-1</span>
+            <div class="col-small-1 col-medium-1 col-3">
+              col-small-1 col-medium-1 col-2
             </div>
           </div>
         </div>
+        <div class="col-small-1 col-medium-3 col-3">
+          col-small-2 col-medium-3 col-3
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-small-4 col-medium-6 col-12">
+      <h2>Nesting at breakpoints (resize to reveal)</h2>
+    </div>
+  </div>
+
+  <div class="u-hide--medium u-hide--large">
+    <div class="row">
+      <div class="col-small-4 col-medium-6 col-12">
+        <h5>Nesting on mobile</h5>
       </div>
     </div>
-    <div class="col-6">
-      <span>.col-6</span>
+
+    <div class="row" css-grid-demo>
+      <div class="col-small-4">
+        col-small-4
+        <div class="row">
+          <div class="col-small-2">
+            col-small-2
+            <div class="row">
+              <div class="col-small-1">
+                col-small-1
+              </div>
+              <div class="col-small-1">
+                col-small-1
+              </div>
+            </div>
+          </div>
+          <div class="col-small-2">
+            col-small-2
+          </div>
+        </div>
+      </div>
+      <div class="col-small-3">
+        col-small-3
+        <div class="row">
+          <div class="col-small-1">
+            col-small-1
+          </div>
+          <div class="col-small-1">
+            col-small-1
+          </div>
+          <div class="col-small-1">
+            col-small-1
+          </div>
+        </div>
+      </div>
+      <div class="col-small-1">
+        col-small-1
+      </div>
+      <div class="col-small-4">
+        <div class="row">
+          <div class="col-small-1">
+            col-small-1
+          </div>
+          <div class="col-small-2">
+            col-small-2
+            <div class="row">
+              <div class="col-small-1">
+                col-small-1
+              </div>
+              <div class="col-small-1">
+                col-small-1
+              </div>
+            </div>
+          </div>
+          <div class="col-small-1">
+            col-small-1
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="u-hide--small u-hide--large">
```

Comment:
Shouldn't this just be a single example demoing the change in grid based on screen size